standup_notes/__main__.py

```diff
@@ -119,6 +124,7 @@ def copy_prev(day: date):
                         copy_text = True
                     if 'Blockers' in line:
                         copy_text = False
+            lines_to_append = get_text(previous_days_note, 't')
```

Comment:
Should this replace all the code above it from the `with open` line to the line right before this?

Reply:
It does replace it, it is now deleted